project-base/app/src/DataFixtures/Demo/SalesRepresentativeDataFixture.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\DataFixtures\Demo;
+
+use Doctrine\Persistence\ObjectManager;
+use Ramsey\Uuid\Uuid;
+use Shopsys\FrameworkBundle\Component\DataFixture\AbstractReferenceFixture;
+use Shopsys\FrameworkBundle\Model\SalesRepresentative\SalesRepresentativeDataFactoryInterface;
+use Shopsys\FrameworkBundle\Model\SalesRepresentative\SalesRepresentativeFacade;
+
+class SalesRepresentativeDataFixture extends AbstractReferenceFixture
+{
+    private const string UUID_NAMESPACE = 'd660b914-5765-42ec-a64a-df9661e6af21';
+
+    public const string SALES_REPRESENTATIVE_JD = 'sales_representative_jd';
+    public const string SALES_REPRESENTATIVE_PS = 'sales_representative_ps';
```

Comment:
I am not sure whether the "JD" and "PS" postfixes are clear enough, I would rather use 1 and 2 :sweat_smile:

Reply:
i was not sure either, now it is changed as you recommended